x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_detail_panel/data_stream_detail_panel.tsx

```diff
@@ -273,15 +298,58 @@ export const DataStreamDetailPanel: React.FunctionComponent<Props> = ({
         toolTip: i18n.translate('xpack.idxMgmt.dataStreamDetailPanel.dataRetentionToolTip', {
           defaultMessage: 'The amount of time to retain the data in the data stream.',
         }),
-        content: getLifecycleValue(lifecycle),
+        content: (
+          <ConditionalWrap
+            condition={isDataStreamFullyManagedByILM(dataStream)}
+            wrap={(children) => <s>{children}</s>}
+          >
+            <>{getLifecycleValue(lifecycle)}</>
+          </ConditionalWrap>
+        ),
         dataTestSubj: 'dataRetentionDetail',
       },
     ];
 
     const managementDetails = getManagementDetails();
     const details = [...defaultDetails, ...managementDetails];
 
-    content = <DetailsList details={details} />;
+    content = (
+      <>
+        {isDataStreamFullyManagedByILM(dataStream) && (
+          <>
+            <EuiCallOut
+              title={i18n.translate(
+                'xpack.idxMgmt.dataStreamsDetailsPanel.editDataRetentionModal.fullyManagedByILMTitle',
+                { defaultMessage: 'This data stream is fully managed by ILM' }
```

Comment:
```suggestion
                { defaultMessage: 'This data stream and its associated indices are managed by ILM' }
```
Mostly trying to disambiguate the meaning of "fully" here to be as explicit as possible.